### DISCUSSION TITLE
Richeditor File not showing

### DISCUSSION THREAD

**Discussion by dingggjc** [filamentphp/filament#12859] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

<img width="944" alt="mao" src="https://github.com/filamentphp/filament/assets/104723506/f0cb5987-b8c3-45ab-9217-62d8afe68367">
i wanna know why the image is not showing in the form, please i really need help

**Comment by leandrocfe** [comment]
did you config `APP_URL` in the .env?
did you run `php artisan storage:link` ?
